docs/api/source/index.rst

```diff
@@ -1,14 +1,103 @@
 :orphan:
 
-==============================
-solid-client-notifications API
-==============================
+====================================================
+@inrupt/solid-client-notifications API Documentation
+====================================================
 
+`@inrupt/solid-client-notifications
+<https://npmjs.com/package/@inrupt/solid-client-notifications>`__ is a
+JavaScript library for requesting and managing access given to an agent for a
+resource. These access grants are represented by Verifiable Credentials, signed
+by an Issuer associated to the Pod server where the resources are hosted.
```

Comment:
```suggestion
<https://npmjs.com/package/@inrupt/solid-client-notifications>`__ is a
JavaScript library for subscribing to notifications sent by a Solid server.
These notifications may be sent synchronously e.g., through WebSockets,
or asynchronously e.g. via WebHooks. ? Notifications are typically sent
when a resource is updated/deleted, or when a child resource is added
to/removed from a container.
```

Reply:
Have updated it, but with a refined version of the text above.